src/strain-config.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const yaml = require('js-yaml');
+const hash = require('object-hash');
+
```

Comment:
maybe describe what a `strain` type is here. use `@typedef` like here: https://github.com/adobe/hypermedia-pipeline/blob/master/src/pipeline.js#L24-L33

or create a `Strain` class....